mainnet/shared/genesis_35.json

```diff
@@ -14,23 +14,23 @@
     "londonBlock": 0,
     "mergeForkBlock": 0,
     "terminalTotalDifficulty": 0,
-    "shanghaiTime": 1681738270
+    "shanghaiTime": 1687969198
   },
   "number": "0x0",
   "nonce": "0x1",
   "difficulty": "0x1",
-  "gasLimit": "0x280de80",#42M
+  "gasLimit": "0x280de80",
   "gasUsed": "0x0",
-  "timestamp": "0x643D34C0",
+  "timestamp": "0x646CE7B0",
   "extraData": "0x416e642074686520776f726c6420626563616d65206f6e65",
   "coinbase": "0x0000000000000000000000000000000000000000",
   "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "alloc": {
-    "0x953534215D31D0f2756b8A855aaFB7E641848c2e": { # FNCE
+    "0x953534215D31D0f2756b8A855aaFB7E641848c2e": {
```

Comment:
4243518/35000000 = 0.1212 => 12.12% != 11.84% as defined on the deposit homepage.
Am I missing something?

Reply:
see above